seed/challenges/01-front-end-development-certification/intermediate-ziplines.json

```diff
@@ -97,9 +108,22 @@
         "Cuando hayas terminado, pulsa el botón de \"I've completed this challenge\" e incluye un enlace a tu CodePen.",
         "Puedes obtener retroalimentación sobre tu proyecto por parte de otros campistas, compartiendolo en nuestra <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Sala de chat para revisión de código</a>. También puedes compartirlo en Twitter y en el campamento de tu ciudad (en Facebook)."
       ],
+      "descriptionFr": [
+        "<strong>Objectif:</strong> Créer une application sur <a href='http://codepen.io' target='_blank'>CodePen.io</a> qui a un fonctionnement similaire à: <a href='http://codepen.io/FreeCodeCamp/full/bELRjV' target='_blank'>http://codepen.io/FreeCodeCamp/full/bELRjV</a>.",
+        "<strong>Règle #1:</strong> Essayez de terminer le projet sans voir le code de l’exemple fourni.",
+        "<strong>Rule #2:</strong> Essayez de répondre aux <a href='https://fr.wikipedia.org/wiki/Récit_utilisateur' target='_blank'>récits d’utilisateurs</a> suivants. Utilisez n’importe quel librairie ou API en ajoutant votre style personnel.",
+        "<strong>Récit d’utilisateur:</strong> Je peux voir la météo de ma région.",
+        "<strong>Récit d’utilisateur:</strong> Je peux voir une icône ou un arrière-plan qui dépendent de la météo (neige, déserts… par exemple).",
+        "<strong> Récit d’utilisateur:</strong> Je peux cliquer sur un bouton pour basculer entre le Celsius et le Fahrenheit.",
+        "On recommande l’utilisation de  <a href='http://openweathermap.org/current#geo' target='_blank'>Open Weather API</a>.  L’utilisation de ce service nécessite la création d’un compte gratuit pour avoir une clé d’utilisation de l’API. Normalement, il faut éviter d’inclure des clés d’API sur CodePen, mais on n’a pas pu trouver un API de météo qui ne nécessite pas une clé.",
+        "N’oubliez pas d’utiliser la méthode  <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si vous êtes bloqué.",
```

Comment:
The wiki link has changed to https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/FreeCodeCamp-Get-Help